Calibration/HcalCalibAlgos/test/python/isoTrackRecoAnalysis_cfg.py

```diff
@@ -32,13 +31,10 @@
 process.load('Calibration.HcalCalibAlgos.HcalIsoTrkAnalyzer_cff')
 process.source = cms.Source("PoolSource", 
                             fileNames = cms.untracked.vstring(
+       'file:/eos/cms/store/group/dpg_hcal/comm_hcal/ISOTRACK/MinBias_AlcaReco_2017D_IsotrkFilter.root',
         #'/store/data/Run2018B/JetHT/ALCARECO/HcalCalIsoTrkFilter-PromptReco-v1/000/317/696/00000/D60EC93B-9870-E811-BAF3-FA163E8DA20D.root',
-#        '/store/mc/Run3Summer19DR/DoublePion_E-50/GEN-SIM-RECO/2021ScenarioNZSRECONoPU_106X_mcRun3_2021_realistic_v3-v2/270000/22481A14-0F65-E046-809A-C03709C76325.root'                        
-#       'file:/afs/cern.ch/work/s/sdey/public/forsunandada/C2F61205-A366-E711-9AFA-02163E01A2B0.root',
-#        '/store/mc/Run3Summer19DRPremix/QCD_Pt-15to7000_TuneCP5_Flat_13TeV_pythia8/GEN-SIM-RECO/2021ScenarioNZSRECO_106X_mcRun3_2021_realistic_v3-v2/210000/0049D816-1F5D-C649-9D36-12A55EF44FE6.root'
-#        '/store/mc/Run3Summer19DRPremix/QCD_Pt-15_IsoTrkFilter_Pt-30_TuneCP5_14TeV-pythia8/GEN-SIM-RECO/2021ScenarioRECO_106X_mcRun3_2021_realistic_v3-v2/30000/9CF8F3E3-B8C7-F84A-BAF2-17AD46769E1E.root'
-                                'root://xrootd.ba.infn.it///store/data/Run2017E/JetHT/ALCARECO/HcalCalIsoTrkFilter-09Aug2019_UL2017-v1/50000/FF19B3B8-39D3-D941-8162-1AA7FB482D48.root'
-
+        #'/store/mc/Run3Summer19DR/DoublePion_E-50/GEN-SIM-RECO/2021ScenarioNZSRECONoPU_106X_mcRun3_2021_realistic_v3-v2/270000/22481A14-0F65-E046-809A-C03709C76325.root'                        
+        #'root://xrootd.ba.infn.it///store/data/Run2017E/JetHT/ALCARECO/HcalCalIsoTrkFilter-09Aug2019_UL2017-v1/50000/FF19B3B8-39D3-D941-8162-1AA7FB482D48.root'
```

Comment:
If you are already touching this file for cosmetic purposes, would you mind removing these lines too? Or explain why they might be needed?